docs/connector-development/tutorials/cdk-tutorial-python-http/4-connection-checking.md

```diff
@@ -26,7 +26,7 @@ class SourcePythonHttpTutorial(AbstractSource):
 ...
 ```
 
-Following the docstring instructions, we'll change the implementation to verify that the input currency is a real currency:
+Following the docstring instructions, we'll change the implementation to verify that the input currency is a real currency (Note: in a real implementation you should write code to connect to the API to validate connectivity and not just validate inputs):
```

Comment:
can you reference OneSignal check_connection it's a simple example that can help people understand it.

Reply:
Sure, done